SESSION_FACILITATOR_GUIDE.md

```diff
@@ -0,0 +1,31 @@
+# 💁🏽 Session Facilitator Guide
+
+This document is intended to provide a helpful starting point or reference for running sessions during summits.
+The new learnings and tools from a summit will ideally be PR'd in when each is concluded.
+
+🎉 Lists are included 🎉
+
+## 📅 in the days leading up to the session
+- [ ] If applicable, port the agenda into [tcq](https://tcq.app/)
```

Comment:
I don't think TCQ is appropriate for every session (although it is great for some).

Some sessions are mostly a person speaking explaining the state of something (for example: streams) and then asking feedback about particular agenda questions.

Some sessions are more open ended where a broader issue is discussed (for example: project velocity) where TCQ works great.

I'd love it if the list reflected this somehow?

Reply:
I'm -1 in putting TCQ as a requirement. I personally have an extremely hard time in using it, and I find my attention span for meetings that use it severely reduced and impaired - I tend to introvert and code during such meetings. I would leave this choice to the facilitator.

Reply:
I'm marking it as optional, mainly so that people know it exists and can discuss whether they'd be down to use it ahead of time.